frontend/src/app/page.tsx

```diff
@@ -0,0 +1,69 @@
+"use client"
+import querystring from "querystring"
+import Image from "next/image"
+import {
+  redirectToSpotifyAuth,
+  requestSpotifyToken,
+  SpotifyAuthConfig,
+} from "./Authorization"
+import { SpotifyCredentials } from "./Credentials"
+import { REDIRECT_URI, SPOTIFY_CLIENT_ID, SPOTIFY_URL } from "../../env"
+
+const AUTH_URL = `${SPOTIFY_URL}authorize?`
+const SCOPE =
+  "playlist-modify-private playlist-modify-public playlist-read-private"
+
+export default function Home() {
+  const authorizeSpotify = () => {
+    if (!SPOTIFY_CLIENT_ID) {
+      throw new Error("Spotify client ID not found")
+    }
+
+    const queryString = querystring.stringify({
+      response_type: "code",
+      client_id: SPOTIFY_CLIENT_ID,
+      scope: SCOPE,
+      redirect_uri: REDIRECT_URI,
+    })
+
+    const authUrl = AUTH_URL + queryString
+
+    window.location.href = authUrl
+  }
+
+  const copyToClipboardAccessToken = () => {
```

Comment:
Nitpick: this two functions look the same. Could we use a single one parametrized by the item name?